packages/coordinator/src/components/V2/Select/Select.tsx

```diff
@@ -30,12 +31,13 @@ export default function Select<T>({
   isDisabled,
   placeholder,
   className,
+  containerClassName,
   isValid = true,
   variant = "standard",
   options,
 }: SelectProps<T>) {
   return (
-    <FormControl>
+    <FormControl className={containerClassName}>
```

Comment:
The components in `/V2` belong to the deprecated coordinator and are going to be deleted.
If you want to use this one, please move it outside this folder

Reply:
done

Reply:
So can you revert the changes here?

Reply:
so I should use V2?

Reply:
done